test/helpers/test-bot.js

```diff
@@ -23,6 +29,11 @@ module.exports = function TestBot (opts = {}) {
   if (opts.name) opts.name = 'ssb-tribes/' + opts.name
 
   const ssb = stack({
+    box2: {
+      legacyMode: true,
+      ...opts.box2
+    },
+    // we don't want testbot to import db2 for us, we want more granularity and control of dep versions
     db1: true,
```

Comment:
maybe patch testbot with a `noDb` option instead of this hack. maybe do that in a different issue/pr

Reply:
made issue https://github.com/ssbc/ssb-tribes/issues/98

Reply:
noDB is a great idea